canonical-webteam/documentation-builder
# Fix snapcraft build

Install git the easier way, to fix the snapcraft build.

QA
--

``` bash
snapcraft cleanbuild
snap install --dangerous documentation-builder_1.6.0_amd64.snap
cd docs/
documentation-builder
```

Check it builds the docs properly.
